IaC/PnP/monitoring/aks/online/aks_secure_baseline/workloads/baseline/traefik.yaml

```diff
@@ -72,6 +72,23 @@ subjects:
   name: traefik-ingress-controller
   namespace: a0008
 ---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: traefik-watch-workloads
+  namespace: ngsa
+  labels:
+    app.kubernetes.io/name: traefik-ingress-ilb
+    app.kubernetes.io/instance: traefik-ingress-ilb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: traefik-ingress-controller
+subjects:
+- kind: ServiceAccount
+  name: traefik-ingress-controller
```

Comment:
why is the ns a0008? should we use a clearer ns?

Reply:
The name was derived from the example used in the previous step (terraform infrastructure setup)